src/pages/resultsView/download/DownloadUtils.ts

```diff
@@ -536,10 +587,21 @@ export function generateDownloadData(
         row.push(rowData.sampleId);
 
         geneSymbols.forEach(gene => {
-            const formattedValue = formatData
-                ? formatData(rowData.alterationData[gene]) // if provided format with the custom data formatter
-                : rowData.alterationData[gene].join(' ') || 'NA'; // else, default format: space delimited join
-
+            let formattedValue: string;
+            if (
+                !isSampleProfiled(
+                    rowData.uniqueSampleKey,
+                    rowData.studyId,
+                    gene
+                )
+            ) {
+                formattedValue = notProfiledString;
+            } else {
+                formattedValue = formatData
+                    ? formatData(rowData.alterationData[gene]) // if provided format with the custom data formatter
+                    : rowData.alterationData[gene].join(' ') ||
```

Comment:
not sure i understand this or statement.    ```rowData.alterationData[gene].join(' ')```
is somehow falsy?

Reply:
i think would be better if you just accomplished with ifs.

Reply:
good catch, this snippet is a nightmare